OvmfPkg/Library/GccLto/liblto-x64.s

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2020, Linaro Ltd. All rights reserved.<BR>
```

Comment:
Please add the copyright info for Intel under this specifically, and revert the year for Linaro:
Copyright (c) 2020, Intel Corporation. All rights reserved.